src/modules/Functions.js

```diff
@@ -0,0 +1,73 @@
+import { modal } from './Dom.js';
+
+function DisplayPopup(data, id, examples) {
```

Comment:
- [ ] kindly remove the extra `id` params since it has no roles in the function for now